tests/attestation_plugin_cert/enc/enc.cpp

```diff
@@ -1,10 +1,15 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
-#include <mbedtls/ctr_drbg.h>
-#include <mbedtls/entropy.h>
+#ifdef OE_USE_OPENSSL
```

Comment:
To verify: is this built and tested in CI/CD both with and without this defined?

Reply:
Yes, both cases are tested. There will be two tests: attestation_plugin_cert_mbedtls (without `OE_USE_OPENSSL` defined) and attestation_plugin_cert_openssl (with `OE_USE_OPENSSL` defined).